andela/wger_vulcan
# Fix pyhon coding style errors on the code base using PEP8 standards

#### What does this PR do?
Fixes the Python coding style warnings/errors 
#### Description of Task to be completed?
Fix PEP8 coding style errors in the codebase
#### How should this be manually tested?
Clone the Repo
```$ pip install pep8``` 
execute the following command
to run the check on a single file``` $pep8 test.py ``` for example
to run the check on all files ``` pep8 *.py ``` 
#### Any background context you want to provide?
The implementation of this feature to enable a uniform style way of writing code making it more readable
#### What are the relevant pivotal tracker stories?
chore Python coding style warnings/errors #142465229
